{
  "comments": [
    {
      "key": {
        "uuid": "29aacfda_7e001c60",
        "filename": "client/isolateserver.py",
        "patchSetId": 8
      },
      "lineNbr": 614,
      "author": {
        "id": 1000487
      },
      "writtenOn": "2018-03-14T14:35:37Z",
      "side": 1,
      "message": "That\u0027s really unfortunate. :/",
      "revId": "ce0685aa4118a056843bbba9ada3742c167fc299",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}